src/components/Dropdown/README.md

```diff
@@ -0,0 +1,71 @@
+# Dropdown
+
+A Dropdown is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button.
+
+---
+
+### Properties
+
+| Prop | Type | Description |
+| ---- | ---- | ----------- |
+| options *| [IDropdownOption] | Items to be presented to the user. |
+| label | string | Text that will be rendered above the dropdown.
+| selectedKey | string | Use to specify an initially selected option. |
+| onChanged | function | Function to call when user changes the selected item. |
+| placeHolder | string | Input placeholder text. Displayed until option is selected. |
```

Comment:
Why did we go with `placeHolder`? The built-in attribute that `<input>` has is `placeholder`.

Reply:
it's Fabric's prop name. there are a few places where we surface Fabric's names directly even when they are not perfect, like the `<Fabric>` component. would you suggest that in general we keep the API surface more similar, or write more abstraction code for better naming or APIs?

Reply:
Ah right, if it's Fabric that's chosen this name, and we're just forwarding the props, then it's not worth changing.